kyo-scheduler/jvm/src/main/scala/kyo/scheduler/Queue.scala

```diff
@@ -1,27 +1,28 @@
-package kyo.scheduler.util
+package kyo.scheduler
 
+import java.lang.invoke.VarHandle
 import java.util.concurrent.atomic.AtomicBoolean
 import kyo.*
 import scala.collection.mutable.PriorityQueue
 
-final private[kyo] class Queue[T](using ord: Ordering[T]) extends AtomicBoolean:
+final private class Queue[T](using ord: Ordering[T]) extends AtomicBoolean:
 
     private val queue = PriorityQueue[T]()
 
-    @volatile private var items = 0
```

Comment:
This is a micro optimization. If `items` is marked as `volatile`, methods like `add` have to use memory barriers to update it within a `modify`/`tryModify` block, which is unnecessary since there's already a write barrier at the end. to release the lock.